components/match_template/src/lib.rs

```diff
@@ -133,78 +131,53 @@ impl MatchTemplate {
     }
 }
 
+#[derive(Debug)]
 enum Substitution {
     Identical(Ident),
-    Map(Ident, Ident),
+    Map(Ident, TokenStream),
 }
 
 impl Parse for Substitution {
     fn parse(input: ParseStream<'_>) -> Result<Self> {
-        let first_ident = input.parse()?;
+        let template_ident = input.parse()?;
```

Comment:
I think this's not an appropriate variable name

Reply:
done